tutor/hooks/__init__.py

```diff
@@ -3,14 +3,7 @@
 
 import typing as t
 
-# These imports are the hooks API
-from . import actions, contexts, filters, priorities
-from .consts import *
+# The imports that follow are the hooks API
+from tutor.core.hooks import clear_all, contexts, priorities
```

Comment:
It seems to me like `contexts` belongs to the `tutor.core.hooks` API, not here in the`tutor.hooks` catalog. Would you agree?

Reply:
Hmmm not quite. This page will actually appear in the "Hook types" section. The `Contexts` class is indeed documented in catalog.rst.

Reply:
Yes, you are correct.